packages/libs/client/src/signing/tests/signWithChainweaver.test.ts

```diff
@@ -36,16 +36,15 @@ describe('signWithChainweaver', () => {
     const { cmd, hash } = unsignedCommand.createCommand();
     const sigs = unsignedCommand.sigs.reduce((sigs, sig, i) => {
       const pubkey = unsignedCommand.signers[i].pubKey;
-      sigs[pubkey] =
-        sig === undefined ? null : sig.sig === undefined ? null : sig.sig;
+      sigs[pubkey] = sig?.sig || null;
       return sigs;
       // eslint-disable-next-line @rushstack/no-new-null
     }, {} as { [pubkey: string]: string | null });
 
     const body = JSON.stringify({ reqs: [{ cmd, hash, sigs }] });
```

Comment:
Shouldn't this have changed from `reqs` to `cmdSigDatas`?

Reply:
Updated